docs/documentation/Introduction.md

```diff
@@ -28,7 +28,7 @@ The `psdk_ros2` wrapper currently supports the following features:
 
 ## Compatibility
 
-The current version supports **PSDK v3.5** and it has been tested and built for **ROS 2 Humble**. For firmware and product compatibility, please check the page [DJI Developer - PSDK](https://developer.dji.com/doc/payload-sdk-tutorial/en/).
+The current version supports **PSDK v3.8** and it has been tested and built for **ROS 2 Humble**. For firmware and product compatibility, please check the page [DJI Developer - PSDK](https://developer.dji.com/doc/payload-sdk-tutorial/en/).
```

Comment:
Here there should be no change. I think it is again a merge issue with main?

Reply:
Apparently it is still v3.5 in main

Reply:
oops!okay then, thank you for noticing it!! I thought this was changed.